app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/WorkspaceTag.scala

```diff
@@ -0,0 +1,11 @@
+package com.azavea.rf.datamodel
+
+import java.util.UUID
+
+case class WorkspaceTag(workspaceId: UUID, TagId: UUID) {
+}
+
+object WorkspaceTag {
+  case class WithRelated(workspadeId: UUID, tag: Tag) {
```

Comment:
Is this necessary? Are we going to expose `WorkspaceTag`s to users directly through the API?

Reply:
It's also a little different from our other `WithRelated` patterns --

usually, there's a "base" class that has some set of things that reference it (like images and thumbnails reference scenes). The relationship goes the other way in this one. Is that because tags are generic now?